core/src/main/java/com/linecorp/armeria/common/logging/RequestLog.java

```diff
@@ -109,10 +109,23 @@ default long responseDurationNanos() {
         return responseEndTimeNanos() - responseStartTimeNanos();
     }
 
+    /**
+     * Returns the amount of time taken since the {@link Request} fully received and until the
+     * {@link Response} processing ended. This property is available only when both
+     * {@link RequestLogProperty#REQUEST_FULLY_RECEIVED} and {@link RequestLogProperty#RESPONSE_END_TIME} are
+     * available.
+     *
+     * @throws RequestLogAvailabilityException if the property is not available yet.
+     * @see RequestLogProperty#RESPONSE_END_TIME
+     */
+    default long requestReceivedDurationNanos() {
+        return responseEndTimeNanos() - requestFullyReceivedTimeNanos();
+    }
+
     /**
      * Returns the amount of time taken since the {@link Request} processing started and until the
      * {@link Response} processing ended. This property is available only when both
-     * {@link RequestLogProperty#REQUEST_START_TIME} and {@link RequestLogProperty#RESPONSE_START_TIME} are
+     * {@link RequestLogProperty#RESPONSE_END_TIME} and {@link RequestLogProperty#RESPONSE_START_TIME} are
```

Comment:
I fixed it because I thought it was typo.

Reply:
I guess the latter one is a type. Because `requestStartTimeNanos()` needs `REQUEST_START_TIME` and `responseEndTimeNanos()` needs `RESPONSE_END_TIME`.

Reply:
You're right 🤦